File: script_linux/21/exemple_1.py
#! /usr/bin/python

def premier (n) :
	i = n - 1
	while (i > 1) :
		if ((n % i) == 0) :
			return (0)
		i = i - 1
	return (1)


j = 1
while (j < 500) :
	if (premier (j)) :
		print j,
	j = j + 1

File: script_linux/22/exemple_1.rb
#! /usr/local/bin/ruby

class Classe

	def initialize (val)
		print "Initialisation de l'instance numero ", val, "\n"
	end

	def methode_1 ()
		print "M‚thode 1\n"
	end

end



instance_1 = Classe . new (1)
instance_1 . methode_1 ()

instance_2 = Classe . new (2)
instance_2 . methode_1 ()
